app/modules/individuals/resources.py

```diff
@@ -364,3 +364,121 @@ def get(self, individual):
         if individual is not None:
             others = individual.get_cooccurring_individuals()
             return others
+
+
+@api.route('/<uuid:individual_guid>/merge')
+@api.response(
+    code=HTTPStatus.NOT_FOUND,
+    description='Individual not found.',
+)
+@api.resolve_object_by_model(Individual, 'individual')
+class IndividualByIDMerge(Resource):
+    """
+    Merge from a list other Individual(s)
+    """
+
+    # permission on target individual is not needed, as user may only have access to other from_individuals
+    # @api.permission_required(
+    #     permissions.ObjectAccessPermission,
+    #     kwargs_on_request=lambda kwargs: {
+    #         'obj': kwargs['individual'],
+    #         'action': AccessOperation.READ,  # only requiring READ here as non-writers basically "request" a merge
+    #     },
+    # )
+    def post(self, individual):
+        from flask_login import current_user
+
+        if not current_user or current_user.is_anonymous:
+            abort(code=401)  # anonymous cannot even request a merge
+
+        req = json.loads(request.data)
+        from_individual_ids = req  # assume passed just list, check for otherwise
+        if isinstance(req, dict):
+            from_individual_ids = req.get('fromIndividualIds', None)
+        if not isinstance(from_individual_ids, list):
+            abort(
+                success=False,
+                message='must pass a list of individuals IDs to merge from',
+                code=500,
+            )
+        if len(from_individual_ids) < 1:
+            abort(
+                success=False,
+                message='list of individuals IDs to merge from cannot be empty',
+                code=500,
+            )
+
+        # NOTE when merge conflict (DEX-514) is addressed, more potential args will be passed in
+
+        meets_minimum = False
+        # for which user does not have edit permissions
+        blocking_encounters = []
+        from_individuals = []
+        for from_id in from_individual_ids:
+            from_indiv = Individual.query.get(from_id)
+            if not from_indiv:
+                abort(
+                    success=False,
+                    message=f'passed from individual id={from_id} is invalid',
+                    code=500,
+                )
+            for enc in from_indiv.encounters:
+                if enc.current_user_has_edit_permission():
+                    meets_minimum = True
+                else:
+                    blocking_encounters.append(enc)
+            from_individuals.append(from_indiv)
+        for enc in individual.encounters:
+            if enc.current_user_has_edit_permission():
+                meets_minimum = True
+            else:
+                blocking_encounters.append(enc)
+
+        if not meets_minimum:
+            AuditLog.security_alert(
```

Comment:
Unauthorised accesses on the frontend are called frontend faults elsewhere in the system. 
I'm not seeing what benefit this is gving us I'm afraid but open to be persuaded. 
But if so the rest of the Frontend faults need asssessing to makke sure they don't need to become security faults

Reply:
mostly what i was going for here is to have some sort of special way of drawing attention to something that "should not happen" from a security point of view.  if this feels more suited to a frontend fault though, i am totally fine with that.

put another way, there should be **no ui option** which gives a user this choice, so if we see this case it might be worth inspecting.   but yeah lemme know if you see this as frontend fault and i will just go with that.

Reply:
That is pretty much what a frontend fault is. 
The frontend has tried to do something that is not permitted.
To put it another way, there should be **no ui option** in our frontend which gives the user this choice but we have an API that is accessible to anyone. In future, a particular group may develop their own API, ok unlikely.
More likely is that someone is trying to grab data from the API and all robustness checks are for that. 
So at the moment, I see all frontend faults as being the frontend has tried to do something that it shouldn't do.  
Admittedly, this apparently has not been used yet which does need resolving

Reply:
security stuff gone. now using `houston_fault`.

Reply:
`frontend_fault()` it is!